### PR TITLE
feat: package and release for Python modules

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

* Refactor modules
* Add targets to package the modules and release the modules.

## Why is it important?

The packages releases can be installed with pip directly from Github

### CODE REVIEW

File: resources/scripts/pytest_apm/Makefile [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+## @help:clean:Remove Python file artifacts.
+.PHONY: clean
+clean:
+	@echo "+ $@"
+	@find . -type f -name "*.py[co]" -delete
+	@find . -type d -name "__pycache__" -delete
+	@find . -name '*~' -delete
+	-@rm -fr *.egg-info build dist $(VENV) bin
+
+	-@rm -fr *.egg-info build dist $(VENV) bin
```
Comment (v1v) [inline resources/scripts/pytest_apm/Makefile:77]: duplicated? 
I'm not familiar with `-@`, what's that?
```suggestion
		-@rm -fr *.egg-info build dist $(VENV) bin
```

Comment (kuisathaverat) [reply resources/scripts/pytest_apm/Makefile:77]: no echo, no fail

Comment (kuisathaverat) [inline resources/scripts/pytest_apm/Makefile:77]:
```suggestion
	-@rm -fr *.egg-info build dist $(VENV) bin
```

File: resources/scripts/pytest_apm/Makefile [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	coverage report -m;
+
+## @help:clean:Remove Python file artifacts.
+.PHONY: clean
+clean:
+	@echo "+ $@"
+	@find . -type f -name "*.py[co]" -delete
+	@find . -type d -name "__pycache__" -delete
+	@find . -name '*~' -delete
+		-@rm -fr *.egg-info build dist $(VENV) bin
```
Comment (kuisathaverat) [inline resources/scripts/pytest_apm/Makefile:75]:
```suggestion
	-@rm -fr *.egg-info build dist $(VENV) bin
```

**Review by v1v — APPROVED** [review]

**Pull request merged**